README.md

```diff
@@ -1,2 +1,178 @@
 # dashboard-template
 A template repository for creating data dashboards with Plotly
+
+## Getting started
+
+### Sign up for GitHub
+
+1.  Sign up for a GitHub account
+1.  Ask the existing team members within the Analytical Dashboards Microsoft Teams to give access to the repository. Make sure to accept the invitation.
+
+**This project will be run in the DAP environment.**
+
+### Setup your local development environment
+
+1.  Set your default browser to Google Chrome - [instructions][Make Chrome your default browser].
+1.  Open Anaconda Navigator via Start menu. **Note:** Anaconda asks if you want to update it, but it won't work.
+1.  Install and launch VS Code (Visual Studio Code) from within the Anaconda Navigator. **Note** after installing VS Code, a Getting Started walkthrough is shown. Click the back button to escape.
+
+[Make Chrome your default browser]: https://support.google.com/chrome/answer/95417?hl=en-GB&co=GENIE.Platform%3DDesktop
+
+### Downloading the code from GitHub
+
+1.  Create a folder on your desktop, for storing source code within if you don't have one already.
+1.  From VS Code open the [Explorer window][explorer_window], the overlapping pages icon on left hand menu. Select the option to "Clone Repository". Click "Clone from GitHub"
+1.  If prompted, authorize with GitHub.
+1.  You should be prompted to enter a repository name. Type "communitiesuk/&lt;Your repository name&gt;". Then click on communitiesuk/&lt;Your repository name&gt;.
+1.  As a destination, select your folder for storing the source code. Select "Sign in with browser" if the GitHub authorisation popup is shown.
+1.  This pulls the code from GitHub to your local folder.
+    Click "Open folder" option, and navigate to your newly created folder containing the repository code.
+1.  Select "Yes, I trust the authors".
+1.  While that is downloading, navigate to the `Git CMD` from the start menu and execute the below commands. Once you have executed the commands close `Git CMD`.
```

Comment:
This only needs to be done once per DAP environment, so _could_ be wasting a very small amount of time, but won't hurt if it gets run multiple times.  I wouldn't worry too much about this.

Reply:
@MattSutton17